az-xdm-instance-upgrade.sh

```diff
@@ -61,7 +74,8 @@ imageOffer=xdm-solution-vm
 if (( ${#versionDigits[@]} == 3 )); then
     if [[ ${versionDigits[2]} = "preview" ]]; then
         planName=${versionDigits[0]}'_'${versionDigits[1]}'_preview'
-        imageVersion=latest
+        imageVersion=0.100.5
```

Comment:
this is for test right ?

Reply:
yep, I still need to use the raw version for now but I will change it to latest once the tests are OK and before the merge